extensions/idp/pkg/backends/cs3/identifier/user.go

```diff
@@ -0,0 +1,63 @@
+package cs3
+
+import (
+	cs3user "github.com/cs3org/go-cs3apis/cs3/identity/user/v1beta1"
+	konnect "github.com/libregraph/lico"
+)
+
+type cs3User struct {
+	u *cs3user.User
+}
+
+func newCS3User(u *cs3user.User) (*cs3User, error) {
+	return &cs3User{
+		u: u,
+	}, nil
+}
+
+func (u *cs3User) Subject() string {
+	return u.u.GetId().GetOpaqueId()
+}
+
+func (u *cs3User) Email() string {
+	return u.u.GetMail()
+}
+
+func (u *cs3User) EmailVerified() bool {
+	return u.u.GetMailVerified()
+}
+
+func (u *cs3User) Name() string {
```

Comment:
As my friend the hound uses to say:
"exported func `Name` should have comment or be unexported"